### PR TITLE
Commit temperature averages kodi

### PR DESCRIPTION
**Pull request by KodzGS**
Please disregard the first commit and review this one mate
Apologies

### CODE REVIEW

File: src/main/java/session3forloops/TemperatureAverageKodi.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package session3forloops;
+
+public class TemperatureAverageKodi {
+
+  public static void main(String[] args) {
+    // Declare and Initialize an Array
+    int[] temperatures = {15, 25, 56, 70, 23, 45, 90};
+    int templength = temperatures.length;
```
Comment (dwilliams-gs) [inline src/main/java/session3forloops/TemperatureAverageKodi.java:8]: Remember to use camel case for variables e.g. `tempLength`

File: src/main/java/session3forloops/TemperatureAverageKodi.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    //For loop
+   /*  for (int i = 0; i < templength; i++) {
+      int temperatureValue = temperatures[i];
+     System.out.printf(temperatureValue + " ");
+    } */
```
Comment (dwilliams-gs) [inline src/main/java/session3forloops/TemperatureAverageKodi.java:14]: Like how you've got an example of using an iterative for loop 👍 
Did you manage to get it summing up the temperatures using this approach also?

**Review by dwilliams-gs — COMMENTED** [review]
Nice work mate 👍 
Just left one very small comment

**Review by dwilliams-gs — APPROVED** [review]

File: src/main/java/session3forloops/TemperatureAverageKodi.java (hunk 2 above)
Comment (KodzGS) [reply src/main/java/session3forloops/TemperatureAverageKodi.java:14]: Did this:
   
 int sum = 0;
    for (int i = 0; i < tempLength; i++) {
      sum += temperatures[i];
    }
    // calculate average
    int averageTemperatureForLoop = sum / tempLength;

File: src/main/java/session3forloops/TemperatureAverageKodi.java (hunk 1 above)
Comment (KodzGS) [reply src/main/java/session3forloops/TemperatureAverageKodi.java:8]: Changed to camel case

    // Declare and Initialize an Array
    int[] temperatures = {15, 25, 56, 70, 23, 45, 90};
    int tempLength = temperatures.length;

**Pull request merged**